src/types/drbTypes.ts

```diff
@@ -9,12 +9,40 @@ export interface DRBQueryParams {
 
 export type DRBFilters = string[]
 
+export interface DRBResults {
+  works?: DRBWork[]
+  totalWorks?: number
+}
+
 export interface DRBWork {
   title?: string
   uuid?: string
+  editions?: Edition[]
+  authors?: Author[]
+  agents?: Agent[]
 }
 
-export interface DRBResultsResponse {
-  works?: DRBWork[]
-  totalWorks?: number
+export interface Edition {
+  title: string
+  items?: EditionItem[]
+}
+
+export interface EditionItem {
+  links: EditionLink[]
+}
+
+export interface Author {
+  name: string
+}
+
+export interface Agent {
+  name: string
+  roles: string[]
+}
+
+export interface EditionLink {
+  link_id: string
```

Comment:
Yeah this is how it comes back from the API.

You're right that it's interesting that typescript doesn't complain here. It complained a while ago when I added underscored names to mapQueryToSearchParams (I added an eslint-disable line since we had to use those values), but I am no longer seeing the errors. Can look into why this changed.

Reply:
Looked into it and I had to add an explicit rule to have eslint throw a warning, not sure why it was working before:

<img width="463" alt="Screenshot 2023-10-11 at 9 26 31 AM" src="https://github.com/NYPL/research-catalog/assets/6901105/38f379fa-7ad8-454d-af04-53fdaef252cb">

Reply:
Ok, I added some explicit naming conventions to eslintrc and disabled for types that can return snake case (api responses and query params)